plugins/camera_sensor/tacamera.py

```diff
@@ -51,9 +51,9 @@ def __init__(self, device='/dev/video0'):
 
     def _on_message(self, bus, message):
         ''' We get a message if a pixbuf is available '''
-        if message.structure is not None:
-            if message.structure.get_name() == 'pixbuf':
-                self.pixbuf = message.structure['pixbuf']
+        if message.get_structure is not None:
+            if message.get_structure.get_name() == 'pixbuf':
+                self.pixbuf = message.get_structure['pixbuf']
```

Comment:
The calls to `message.get_structure` would return the function object and not the structure of the message as intended, it should be `message.get_structure()`.

Reply:
This is probably fixed in ad48974